ios/Capacitor/Capacitor/Plugins/StatusBar.swift

```diff
@@ -35,12 +35,25 @@ public class CAPStatusBarPlugin: CAPPlugin {
     call.unimplemented()
   }
   
+  func setAnimation(_ call: CAPPluginCall) {
+    let animation = call.getString("animation", "SLIDE")
+    if animation == "FADE" {
+      bridge.setStatusBarAnimation(.fade)
+    } else if animation == "SLIDE" {
+      bridge.setStatusBarAnimation(.slide)
+    } else if animation == "NONE" {
+      bridge.setStatusBarAnimation(.none)
+    }
```

Comment:
If the use enters an invalid value, it will use latest animation set instead of the "SLIDE" default.
Change it to set it to `.slide` if the value is not `"NONE"` nor `"FADE"`

```suggestion
    } else if animation == "NONE" {
      bridge.setStatusBarAnimation(.none)
    } else {
      bridge.setStatusBarAnimation(.slide)
    }
```

Reply:
👍